cohorts/count.py

```diff
@@ -57,9 +59,13 @@ def count_func(sample):
         return np.nan
     return count(cohort, count_func, count_col="neoantigen_count")
 
-def count(cohort, count_func, count_col):
-    df = cohort.clinical_dataframe.copy()
-    df[count_col] = df[cohort.clinical_dataframe_id_col].map(count_func)
+def count(cohort, count_func, count_col, group_by="patient"):
+    verify_group_by(group_by)
+
+    id_col = "patient_id" if group_by == "patient" else "sample_id"
```

Comment:
Some docs or comments on the group_by options here? From the PR description I'm gathering this separates samples like tumor and normal from a patient who has both?